docs/how-to/download-real-time-sources.rst

```diff
@@ -0,0 +1,116 @@
+---------------------------------------------
+
+.. tab-set::
+    :sync-group: release
+
+    .. tab-item:: LTS releases (22.04, 24.04, etc.)
+        :sync: lts
+
+        .. note::
+            LTS releases of Ubuntu require Ubuntu Pro to download their respective real-time kernel source code.
+            See the `Ubuntu Pro documentation <https://documentation.ubuntu.com/pro/>`_ for more information.
+
+        If you've already enabled the real-time kernel, you don't need to do anything in this step.
+        You can verify the ``realtime-kernel`` service is activated with ``sudo pro status``.
+
+        Otherwise, enable access to the real-time kernel source packages with the following command:
+
+        .. code-block:: shell
+
+            sudo pro enable realtime-kernel --access-only
+
+
+    .. tab-item:: Interim releases (24.10, etc.)
+        :sync: interim
+        
+        The real-time kernel source is freely available in the 
+        `universe repository <https://packages.ubuntu.com/oracular/linux-realtime>`_ for interim releases. 
+        No actions are required for this step!
+    
+
+Enable downloading source packages with apt
+--------------------------------------------
+
+apt is used to download the real-time kernel source. We need to enable the source repositories (``deb-src``) to allow ``apt`` to download real-time kernel source packages from its archives.
+
+.. tab-set::
+    :sync-group: release
+
+    .. tab-item:: 22.04 LTS and older
+        :sync: lts
+
+        Open the ``/etc/apt/sources.list`` file in a text editor (requires elevated permissions) and :spellexception:`uncomment` (delete the leading ``#``) the line which includes ``deb-src http://archive.ubuntu.com/ubuntu/ <release> main``. 
+
+        For example, for Ubuntu 22.04 LTS (Jammy Jellyfish), only :spellexception:`uncomment` line number 4:
+
+        .. code-block:: debsources
+           :emphasize-lines: 4
+           :linenos:
+
+           # See http://help.ubuntu.com/community/UpgradeNotes for how to upgrade to
+           # newer versions of the distribution.
+           deb http://archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse
+           deb-src http://archive.ubuntu.com/ubuntu/ jammy main restricted universe multiverse
+
+           deb http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted universe multiverse
+           # deb-src http://archive.ubuntu.com/ubuntu/ jammy-updates main restricted universe multiverse
+
+        .. caution::
+            Be sure to :spellexception:`uncomment` only the line which includes ``main`` after the release code-name.
+
+    .. tab-item:: 24.04 LTS and newer
+        :sync: lts
+
+        Use the following command to add ``deb-src`` to the :file:`ubuntu-realtime-kernel.sources` file:
+
+        .. code-block:: shell
+            
+            sudo sed -i '2s/^Types: deb$/Types: deb deb-src/' /etc/apt/sources.list.d/ubuntu-realtime-kernel.sources
```

Comment:
Why using sed to add the deb-src but not for uncommenting that line for 22.04? 

I suggest not using sed at all and simply telling what needs be added with an example. This makes it easier to understand what is being done and to revert it.